spring-kafka/src/main/java/org/springframework/kafka/retrytopic/ListenerContainerFactoryConfigurer.java

```diff
@@ -158,6 +160,20 @@ public ListenerContainerFactoryConfigurer(KafkaConsumerBackoffManager kafkaConsu
 		return new RetryTopicListenerContainerFactoryDecorator(factory, configuration, false);
 	}
 
+	/**
+	 * Set a {@link BackOff} to be used with blocking retries.
+	 * You can specify the exceptions to be retried using the method
+	 * {@link org.springframework.kafka.listener.ExceptionClassifier#addRetryableExceptions(Class[])}
+	 * By default, no exceptions are retried via blocking.
```

Comment:
We should add something like "If the backoff execution returns STOP (retries are exhausted), the record will then go to the next retry topic (if present)."

Reply:
Yes, I think the aggregated behavior of blocking and non-blocking can be better documented in general. I'll see what I can come up with.